modules/Core/classes/Events/RenderContentEditEvent.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+class RenderContentEditEvent extends AbstractEvent {
+
+    public string $content;
+    public bool $skip_purify;
+
+    public function __construct(string $content, bool $skip_purify = false) {
```

Comment:
```suggestion
    public function __construct(string $content, bool $skip_purify) {
```

Should we make this mandatory? Otherwise I know from personal experience it's super easy to forget about 🙈 
Can we also add a phpdoc to indicate exactly what it means for the purification to happen?

Reply:
Should stay false by default, The user should clearly know what they're doing if they skip, Anyways i did add phpdoc

Sam also agrees It should be a conscious decision to skip